tests/unit/aiplatform/test_initializer.py

```diff
@@ -69,9 +70,17 @@ def test_init_location_with_invalid_location_raises(self):
         with pytest.raises(ValueError):
             initializer.global_config.init(location=_TEST_INVALID_LOCATION)
 
-    def test_init_experiment_sets_experiment(self):
-        initializer.global_config.init(experiment=_TEST_EXPERIMENT)
-        assert initializer.global_config.experiment == _TEST_EXPERIMENT
+    # TODO(nachocano) Mock the call to set_experiment
+    # def test_init_experiment_calls_metadata_service(self):
```

Comment:
@sasha-gitg how can I mock the call to set_experiment?

Reply:
Preferably mock the service calls that get called underneath.

Reply:
OK, ended up mocking the the set calls. Once we create the UTs for the metadata_service, I think we can mock what they actually end up calling...